Bookish/Controllers/BookController.cs

```diff
@@ -0,0 +1,86 @@
+using System.Diagnostics;
+using Microsoft.AspNetCore.Mvc;
+using Bookish.Models;
+using Microsoft.AspNetCore.Authorization.Infrastructure;
+
+namespace Bookish.Controllers;
+
+public class BookController : Controller
+{
+    public IActionResult Catalogue()
+    {
+        var books = EditBooks.GetBookList();
+        return View(books);
+    }
+    
+    [HttpGet]
+    public IActionResult AddBook()
+    {
+        return View();
+    }
+    
+    [HttpPost]
+    public ActionResult AddBook(BookModel newBook)
+    {
+        EditBooks.InsertBooks(newBook);
+        return View(newBook);
+    }
+    [HttpGet]
+    public ActionResult EditBook(int id)
+    {
+        BookModel updateBook = EditBooks.GetBookFromId(id);
+        return View(updateBook);
+    }
+    
+    [HttpPost]
+    public ActionResult EditBook(BookModel replaceBook)
+    {
+        EditBooks.AlterBooks(replaceBook.Id, replaceBook);
+        return RedirectToAction(nameof(SearchBooksResults));
+    }
+    
+    [HttpPost]
+    public ActionResult DeleteBook(BookModel lostBook)

```

Comment:
Does a book have to be lost for it to be deleted? What if the library just removes a book because it's outdated? Could the variable name be better?